packages/size-limit/test/run.test.js

```diff
@@ -92,7 +92,7 @@ async function error(cwd, args) {
 
 describe(`run`, () => {
   beforeEach(() => {
-    jest.useFakeTimers()
+    jest.useFakeTimers('legacy')
```

Comment:
Tests fail without `'legacy'` wih `ReferenceError: ClockDate is not defined`. It looks like webpack tries to bundle the Sinon mock timers:

```
  at eval (eval at Object.<anonymous> (/node_modules/webpack/lib/serialization/ObjectMiddleware.js:128:17), <anonymous>:3:1)
        at Object.<anonymous> (/node_modules/webpack/lib/serialization/ObjectMiddleware.js:128:47)
        at Runtime._execModule (/node_modules/jest-runtime/build/index.js:1646:24)
        at Runtime._loadModule (/node_modules/jest-runtime/build/index.js:1185:12)
        at Runtime.requireModule (/node_modules/jest-runtime/build/index.js:1009:12)
        at Runtime.requireModuleOrMock (/node_modules/jest-runtime/build/index.js:1210:21)
        at /node_modules/webpack/lib/util/serialization.js:19:2
        at /node_modules/webpack/lib/util/memoize.js:22:13
        at Object.get register [as register] (/node_modules/webpack/lib/util/serialization.js:56:10)
        at Object.<anonymous> (/node_modules/webpack/lib/util/makeSerializable.js:7:9)
``` 
There's a few issues in the Jest repository related to the new timer implementation: https://github.com/facebook/jest/issues?q=is%3Aissue+is%3Aopen+timer I don't know whether we should file a bug to Jest or we are abusing the mock system in some way.